app/src/main/java/com/asfoundation/wallet/ui/BaseActivity.java

```diff
@@ -105,4 +108,11 @@ protected void onActivityResult(int requestCode, int resultCode, @Nullable Inten
       listener.onActivityResult(requestCode, resultCode, data);
     }
   }
+
+  private void hideKeyboard() {
```

Comment:
We already have an utils method that hides the keyboard . Shouldn't we reuse it?

=> Class KeyboardUtils